apps/src/code-studio/components/AssetUploader.jsx

```diff
@@ -38,15 +40,16 @@ export default class AssetUploader extends React.Component {
           onUploadDone={uploadDone}
           onUploadError={this.props.onUploadError}
         />
-        <button
+        <Button
           onClick={this.fileUploadClicked}
           className="share"
           id="upload-asset"
           disabled={!this.props.uploadsEnabled}
-        >
-          <i className="fa fa-upload" />
-          &nbsp;Upload File
-        </button>
+          icon="upload"
+          text="Upload File"
```

Comment:
Do we need to i18n any of these strings?

Reply:
Ah! Good catch! Thanks!